### PR TITLE
Added camera stream from robot to the web app

### PR DESCRIPTION
**Pull request by elishajcat**

### CODE REVIEW

File: frontend/src/pages/Control.jsx [hunk 1]
```diff
@@ -12,6 +12,8 @@ import Chip from "@mui/material/Chip";
 
 import { useState, useEffect } from "react";
 
+const STREAM_URL = "http://172.17.76.72:5000/stream.mjpg";
```
Comment (deestroy) [inline frontend/src/pages/Control.jsx:15]: change to a dynamic IP with evan?

File: robot-endpoint/stream_camera.py [hunk 2]
```diff
@@ -0,0 +1,131 @@
...
+        self.wfile.write(jpeg.tobytes())
+        self.wfile.write(b'\r\n')
+
+
+class StreamingServer(socketserver.ThreadingMixIn, HTTPServer):
+    allow_reuse_address = True
+    daemon_threads = True
+
+def opencv_camera_stream():
+    cap = cv2.VideoCapture(0)
```
Comment (deestroy) [inline robot-endpoint/stream_camera.py:90]: did it work without the GStreamer?

File: frontend/src/pages/Control.jsx (hunk 1 above)
Comment (elishajcat) [reply frontend/src/pages/Control.jsx:15]: are we doing that for this demo?

File: robot-endpoint/stream_camera.py (hunk 2 above)
Comment (elishajcat) [reply robot-endpoint/stream_camera.py:90]: yeah this was my latest changes from last night and it worked

**Review by deestroy — APPROVED** [review]

**Pull request merged**